main.go

```diff
@@ -138,9 +144,37 @@ func main() {
 		attachProfiler(routes)
 	}
 
+	putAddr := fmt.Sprintf(":%v", DEFAULT_PUT_PORT)
+	if putPort := os.Getenv("LIVELOG_PUT_PORT"); putPort != "" {
+		pp, err := strconv.Atoi(putPort)
+		if err != nil {
+			debug("env var LIVELOG_PUT_PORT is not a number (%v)", putPort)
+			os.Exit(64)
+		}
+		if pp < 0 || pp > 65535 {
+			debug("env var LIVELOG_PUT_PORT is not between [0, 65535] (%v)", pp)
+			os.Exit(65)
+		}
+		putAddr = ":" + putPort
+	}
+
+	getAddr := fmt.Sprintf(":%v", DEFAULT_GET_PORT)
+	if getPort := os.Getenv("LIVELOG_GET_PORT"); getPort != "" {
+		gp, err := strconv.Atoi(getPort)
+		if err != nil {
+			debug("env var LIVELOG_GET_PORT is not a number (%v)", getPort)
+			os.Exit(66)
+		}
+		if gp < 0 || gp > 65535 {
+			debug("env var LIVELOG_GET_PORT is not between [0, 65535] (%v)", gp)
+			os.Exit(67)
+		}
+		getAddr = ":" + getPort
+	}
+
```

Comment:
Can we avoid duplicated code by introducing a helper function more or less like:

```go
func getPortNumber(envVar string) (int, error)
```